### PR TITLE
Add service_enable option

### PR DESCRIPTION
**Pull request by sergiik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,6 +121,15 @@ Ensure whether the service is running or stopped. If you're passing `absent` to
 |-----|--------|
 | Enum['running', 'stopped'] | `running` |
 
+#### service_enable
+
+Set whether the service is enabled or not. If you're passing `absent` to
+`ensure`, please also pass `false` to `service_enable`.
```
Comment (b4ldr) [inline README.md]: we should enforce this either by throwing an error or always setting `$service_enable = false if ensure => absent`

**Comment by bastelfreak** [comment]
Hi @sergiik, thanks for the PR. Can you take a look at the inline comment and add a unit test for this?

**Comment by sergiik** [comment]
> Hi @sergiik, thanks for the PR. Can you take a look at the inline comment and add a unit test for this?

Ok. Will do

**Comment by ekohl** [comment]
Closing for inactivity and merge conflicts. Please open a new PR if you're still interested in getting this merged.

**Pull request closed without merging**